cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -857,7 +860,8 @@ pub mod bridging {
 			/// (initially was calculated by test `OutboundQueue::calculate_fees` - ETH/ROC 1/400 and fee_per_gas 20 GWEI = 2200698000000 + *25%)
 			/// Needs to be more than fee calculated from DefaultFeeConfig FeeConfigRecord in snowbridge:parachain/pallets/outbound-queue/src/lib.rs
 			/// Polkadot uses 10 decimals, Kusama and Rococo 12 decimals.
-			pub const BridgeHubEthereumBaseFeeInROC: u128 = 2_750_872_500_000;
+			pub const DefaultBridgeHubEthereumBaseFeeInROC: u128 = 2_750_872_500_000;
```

Comment:
Need the non-storage const for the tests as the storage modifier causes tests to panic as there is no underlyiing storage provider.